.github/workflows/translations_push.yml

```
@@ -0,0 +1,29 @@
+name: 🌎 Push translations to transifex
```

```suggestion
name: 🌎 Push translations to Transifex
```

Just looking for consistency in writing the name.